src/uu/tail/src/args.rs

```diff
@@ -228,22 +228,15 @@ impl Settings {
         };
 
         if let Some(source) = matches.get_one::<String>(options::SLEEP_INT) {
-            // Advantage of `fundu` over `Duration::(try_)from_secs_f64(source.parse().unwrap())`:
-            // * doesn't panic on errors like `Duration::from_secs_f64` would.
-            // * no precision loss, rounding errors or other floating point problems.
-            // * evaluates to `Duration::MAX` if the parsed number would have exceeded
-            //   `DURATION::MAX` or `infinity` was given
-            // * not applied here but it supports customizable time units and provides better error
-            //   messages
-            settings.sleep_sec = match DurationParser::without_time_units().parse(source) {
-                Ok(duration) => SaturatingInto::<Duration>::saturating_into(duration),
+            settings.sleep_sec = match parse_time::from_str(source, false) {
```

Comment:
can probably be simplified this way:

settings.sleep_sec = parse_time::from_str(source, false)
    .map_err(|_| UUsageError::new(1, format!("invalid number of seconds: '{source}'")))?;